Makefile

```diff
@@ -1,4 +1,5 @@
 
+BROWSERIFY := node_modules/.bin/browserify
```

Comment:
I guess we could do something like `PATH := node_modules/.bin:$(PATH)` to simplify the code (slightly). Seeing as you'd also prefer a local dep. over a global one anyway.